eo-maven-plugin/src/test/java/org/eolang/maven/PullMojoTest.java

```diff
@@ -291,4 +301,13 @@ private static boolean exists(final Path temp, final String source) {
     private static String path(final String source) {
         return String.format(source, PullMojo.DIR);
     }
+
+    /**
+     * Format given a source path.
+     * @param source Source path as object name.
+     * @return Formatted source path.
+     */
+    private static String path(final ObjectName source) {
```

Comment:
@maxonfjvipon Since this method uses overloaded `path` method it's better to place it above that method:
> In [Clean Code](https://rads.stackoverflow.com/amzn/click/com/0132350882), Robert C. Martin advises coders to always put member variables at the top of the class (constants first, then private members) and methods should be ordered in such a way so that they read like a story that doesn't cause the reader to need to jump around the code too much. This is a more sensible way to organize code rather than by access modifier

https://stackoverflow.com/a/1760877/10423604

It is something similar with cascade constructors.